test/Microsoft.AspNetCore.Authentication.Test/OpenIdConnect/OpenIdConnectTests.cs

```diff
@@ -294,6 +294,31 @@ public class OpenIdConnectTests
             Assert.Equal("Cannot redirect to the end session endpoint, the configuration may be missing or invalid.", exception.Message);
         }
 
+        // Test Cases for converting OpenIdConnectOptions.MaxAge to an appropriate max_age 
+        // parameter for the authenticate request.
+        [Fact]
+        public void MaxAgeConversion()
+        {
+            // max_age not set for default value
+            var options = new OpenIdConnectOptions();
+            Assert.Null(OpenIdConnectHelper.GetMaxAgeParameterFromTimeSpan(options.MaxAge));
+
+            // max_age not set for values less than 1 second
+            Assert.Null(OpenIdConnectHelper.GetMaxAgeParameterFromTimeSpan(TimeSpan.FromSeconds(0.999)));
+            Assert.Null(OpenIdConnectHelper.GetMaxAgeParameterFromTimeSpan(new TimeSpan(0, 0, 0)));
+            Assert.Null(OpenIdConnectHelper.GetMaxAgeParameterFromTimeSpan(TimeSpan.FromMinutes(-10)));
```

Comment:
IMHO it would be better to throw an exception when validating the OIDC configuration. You should consider adding a `if (options.MaxAge != null && options.MaxAge.Value < TimeSpan.Zero)` check in `OpenIdConnectPostConfigureOptions.PostConfigure`.

https://github.com/aspnet/Security/blob/dev/src/Microsoft.AspNetCore.Authentication.OpenIdConnect/OpenIdConnectPostConfigureOptions.cs#L31